presto-docs/src/main/sphinx/release/release-0.240.rst

```diff
@@ -0,0 +1,64 @@
+=============
+Release 0.240
+=============
+
+General Changes
+_______________
+* Fix LambdaDefinitionExpression canonicalization did not handle CAST.
```

Comment:
What is the user impact?  Needs to be rephrased

Reply:
@rongrong This is from #14912  . Can you rephrase this and comment here. I can update the notes.

Reply:
`Fix compiler error due to incorrect LambdaDefinitionExpression canonicalization`.